src/types.ts

```diff
@@ -80,7 +83,6 @@ export type EasyCropRef = {
 };
 
 export type EasyCropProps = {
-  cropperRef: ForwardedRef<Cropper>;
```

Comment:
Removed the cropper ref since it wasn't needed in our specific use case and was causing TS complications